config/i18n.json

```diff
@@ -554,6 +554,8 @@
     "SelectModsCountActivityMods": "{{selected}}/{{maxSelectable}} Activity Mods",
     "SelectExotic": "Select exotic",
     "SelectMods": "Select Mods",
+    "AddStack": "Add another copy of this mod",
+    "RemoveStack": "Remove a copy of this mod",
```

Comment:
nit: When there are 0 -> "Add this mod", when there are 1 -> "Remove this mod". Probably not worth the complexity but something I noticed.

Reply:
https://www.i18next.com/translation-function/plurals

Reply:
Yes my exact thought.

Reply:
Not worth it just for the titles on these buttons I think